mempool/v1/reactor.go

```diff
@@ -302,14 +299,13 @@ func (memR *Reactor) broadcastTxRoutine(peer p2p.Peer) {
 				// record that we have sent the peer the transaction
 				// to avoid doing it a second time
 				memTx.SetPeer(peerID)
+				schema.WriteMempoolTx(
+					memR.traceClient,
+					peer.ID(),
+					memTx.tx.Hash(),
+					schema.Upload,
+				)
 			}
-			schema.WriteMempoolTx(
-				memR.traceClient,
-				peer.ID(),
-				memTx.tx,
-				schema.TransferTypeUpload,
-				schema.V1VersionFieldValue,
-			)
```

Comment:
these were moved to only write if the send was successful

Reply:
Makes sense